test/tests/core/Mat/imgprocTests.js

```diff
@@ -1479,5 +1479,64 @@ module.exports = (getTestImg) => {
         expectOutput: makeExpectOutput(cv32fc6)
       });
     });
+
+    describe('goodFeaturesToTrack', () => {
+      generateAPITests({
```

Comment:
Shouldn't this test suite be wrapped in `if(cv.version.minor >= 4) { ... }`? I wonder how the tests passed on this for lower CV versions on Travis.

Reply:
related to: https://github.com/justadudewhohacks/opencv4nodejs/pull/217#issuecomment-375768389